libcloud/compute/drivers/gce.py

```diff
@@ -4201,7 +4205,7 @@ def create_node(
             ex_can_ip_forward, ex_disks_gce_struct, ex_nic_gce_struct,
             ex_on_host_maintenance, ex_automatic_restart, ex_preemptible,
             ex_subnetwork, ex_labels, ex_accelerator_type,
-            ex_accelerator_count)
```

Comment:
To avoid potential bugs it would probably be safer to use keyword instead of positional argumentshere  in the future, but this also works for now since other arguments are already positional.

Reply:
I definitely agree, and actually was thinking about fixing it, but then decided against it so as to keep the scope of this PR small.